pkg/gameservers/controller_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	"agones.dev/agones/pkg/cloudproduct/generic"
 	agtesting "agones.dev/agones/pkg/testing"
+	agonesruntime "agones.dev/agones/pkg/util/runtime"
```

Comment:
this is conventionally not aliased and kept as `runtime` - is that a problem? I don't think I've seen it aliased in our code.

Reply:
If I do `runtime "agones.dev/agones/pkg/util/runtime"` there's a conflict with `"k8s.io/apimachinery/pkg/runtime"` which is already declared and I cannot remove that library because it'll trigger many undefined errors.

Reply:
Oh, I missed that conflict. It looks like elsewhere when there's a conflict we import it as `k8sruntime`: https://github.com/search?q=repo%3Agoogleforgames%2Fagones%20%22k8s.io%2Fapimachinery%2Fpkg%2Fruntime%22&type=code .. I would be inclined to switch that one instead.

I know this sounds nitpicky but I've often found it's an important readability thing, otherwise a reader conversat in our habits may get confused.

Reply:
I see, makes sense, thanks! Nevertheless, since i'm moving the implementation to GKE `cloudproduct` I'll add a unit test inside gke_test.